mowgli_etl/pipeline/wdc/wdc_offers_corpus_entry.py

```diff
@@ -29,7 +29,9 @@ class __WdcOffersCorpusEntryIdentifier:
     id: int
     identifiers: Optional[List[__WdcOffersCorpusEntryIdentifier]]
     key_value_pairs: Optional[dict] = field(metadata=config(field_name="keyValuePairs"))
-    spec_table_content: Optional[str] = field(metadata=config(field_name="specTableContent"))
+    spec_table_content: Optional[str] = field(
```

Comment:
You shouldn't need to specify this field metadata. The @dataclass_json decorator can be:

```
@dataclass_json(letter_case=LetterCase.CAMEL)  # now all fields are encoded/decoded from camelCase
```

See the docs.